### PR TITLE
Confirmed support for Django 4.1

### PR DESCRIPTION
**Pull request by hramezani**

### CODE REVIEW

**Comment by timgraham** [comment]
In this case it's "Confirmed support... (no code changes were required)" (see similar past release notes). You should also add a placeholder heading in the changelog for unreleased changes as done in the past.

File: docs/changelog.rst [hunk 1]
```diff
@@ -1,6 +1,11 @@
 Changelog
 =========
 
+5.2 (unreleased)
```
Comment (timgraham) [inline docs/changelog.rst:4]: I'd use X.Y since it's uncertain what the next version might be.

**Review by timgraham — APPROVED** [review]

File: docs/changelog.rst (hunk 1 above)
Comment (hramezani) [reply docs/changelog.rst]: Yes, You are right. just updated!

**Comment by hramezani** [comment]
Thanks @timgraham for review :+1:

**Pull request merged**